woocommerce/woocommerce-gateway-payfast
# Updated plugins's main file to match with plugin's slug.

### All Submissions:



---

### Changes proposed in this Pull Request:
PR updates the plugin's main file to match the plugin's slug and keeps the old file for backward compatibility. 

Closes #178 

**Note:** @dkotter I have noticed that `gateway-payfast.php` is [used](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/class-wc-admin-setup-wizard.php#L1508) in WooCommece Core. So, better to confirm with the WooCommerce Team if we need to update this somewhere else as well before releasing the plugin.

### Steps to test the changes in this Pull Request:
1. Install and activate the plugin with the `trunk` branch.
1. Switch to this PR branch.
1. Go to the plugins list and make sure the plugin is activated and working as expected.

### Changelog entry
> Dev -  Updated the main file of the plugin to match the plugin's slug.
